Algebraic_kernel_d/include/CGAL/Algebraic_kernel_d/Shear_controller.h

```diff
@@ -89,13 +91,15 @@ namespace CGAL {
 	Int get_new_shear_factor() {
 	  CGAL_assertion(int(this->bad_shears.size())<m_max);
 	  while(true) {
-	    Int s = Int(rand()%m_max)+1;
+	    Int s = Int((rng.get_int(0,(std::numeric_limits<int>::max)())%m_max )+1);
```

Comment:
Why not use that simpler expression?
```c++
Int s = Int(rng.get_int(1,m_max));
```

Reply:
That's what I did first, but it does not work for `CORE::Big_int`.